packages/onyxsdk_pen/lib/onyxsdk_pen_area.dart

```diff
@@ -21,6 +24,9 @@ class OnyxSdkPenArea extends StatefulWidget {
   /// is still writing, which will make the screen get stuck in a half-drawn
   /// state.
   final Duration refreshDelay;
+  final int strokeStyle;
```

Comment:
This should be an enum arg too like in kotlin: e.g. `strokeStyle: StrokeStyle.fountainPen`